.github/workflows/manually-push-to-ghcr.yml

```diff
@@ -0,0 +1,30 @@
+name: Manually Build Push Image to GitHub Registry
+
+on: workflow_dispatch
+
+jobs:
+  manually-triggered-build-push-to-ghcr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Login to GitHub Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build and Push Image to GitHub Registry
+        uses: docker/build-push-action@v2
+        with:
+          context: ./CI 
+          file: ./CI/Dockerfile
+          builder: ${{ steps.buildx.outputs.name }}
+          push: true
+          tags: ghcr.io/${{ github.repository_owner }}/stm32-cubeide-headless:latest
```

Comment:
I'm not sure this will work if the username has uppercase letters in it (like mine). Could you check this please?

Reply:
Ok, I will and I'll get back to you.

Reply:
@Niuslar Yup, you were right, it didn't work. I lowercased the username.

Reply:
Thanks, let's merge then.